### ISSUE TITLE
Crash report

### ISSUE THREAD

**Issue by devstopfix** [devstopfix/elixoids#23]
```
=ERROR REPORT==== 11-Jul-2016::12:53:38 ===
Ranch listener http had connection process started with cowboy_protocol:start_link/4 at <0.3550.0> exit with reason: {[{reason,badarg},{mfa,{'Elixir.WebsocketShipHandler',websocket_terminate,3}},{stacktrace,[{erlang,send,[game,{'$gen_cast',{remove_player,<<"OCA">>}}],[]},{'Elixir.GenServer',do_send,2,[{file,"lib/gen_server.ex"},{line,639}]},{'Elixir.WebsocketShipHandler',websocket_terminate,3,[{file,"lib/websocket_ship_handler.ex"},{line,53}]},{cowboy_websocket,handler_terminate,4,[{file,"src/cowboy_websocket.erl"},{line,750}]},{cowboy_websocket,handler_call,7,[{file,"src/cowboy_websocket.erl"},{line,641}]},{cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,442}]}]},{req,[{socket,#Port<0.14035>},{transport,ranch_tcp},{connection,keepalive},{pid,<0.3550.0>},{method,<<"GET">>},{version,'HTTP/1.1'},{peer,{{127,0,0,1},50875}},{host,<<"127.0.0.1">>},{host_info,undefined},{port,8065},{path,<<"/ship/OCA">>},{path_info,[<<"OCA">>]},{qs,<<>>},{qs_vals,undefined},{bindings,[]},{headers,[{<<"host">>,<<"127.0.0.1:8065">>},{<<"upgrade">>,<<"websocket">>},{<<"connection">>,<<"Upgrade">>},{<<"sec-websocket-key">>,<<"r/TyIBb1ETjxdi+5kOD9fg==">>},{<<"sec-websocket-version">>,<<"13">>}]},{p_headers,[{<<"upgrade">>,[<<"websocket">>]},{<<"connection">>,[<<"upgrade">>]}]},{cookies,undefined},{meta,[{websocket_version,13},{websocket_compress,false}]},{body_state,waiting},{buffer,<<>>},{multipart,undefined},{resp_compress,false},{resp_state,done},{resp_headers,[]},{resp_body,<<>>},{onresponse,undefined}]},{state,<<"OCA">>},{terminate_reason,{error,handler}}],[{cowboy_websocket,handler_terminate,4,[{file,"src/cowboy_websocket.erl"},{line,752}]},{cowboy_websocket,handler_call,7,[{file,"src/cowboy_websocket.erl"},{line,641}]},{cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,442}]}]}

=ERROR REPORT==== 11-Jul-2016::12:53:38 ===
Error in process <0.3550.0> with exit value:
{[{reason,badarg},
  {mfa,{'Elixir.WebsocketShipHandler',websocket_terminate,3}},
  {stacktrace,
      [{erlang,send,[game,{'$gen_cast',{remove_player,<<"OCA">>}}],[]},
       {'Elixir.GenServer',do_send,2,[{file,"lib/gen_server.ex"},{line,639}]},
       {'Elixir.WebsocketShipHandler',websocket_terminate,3,
           [{file,"lib/websocket_ship_handler.ex"},{line,53}]},
       {cowboy_websocket,handler_terminate,4,
           [{file,"src/cowboy_websocket.erl"},{line,750}]},
       {cowboy_websocket,handler_call,7,
           [{file,"src/cowboy_websocket.erl"},{line,641}]},
       {cowboy_protocol,execute,4,
           [{file,"src/cowboy_protocol.erl"},{line,442}]}]},
  {req,
      [{socket,#Port<0.14035>},
       {transport,ranch_tcp},
       {connection,keepalive},
       {pid,<0.3550.0>},
       {method,<<"GET">>},
       {version,'HTTP/1.1'},
       {peer,{{127,0,0,1},50875}},
       {host,<<"127.0.0.1">>},
       {host_info,undefined},
       {port,8065},
       {path,<<"/ship/OCA">>},
       {path_info,[<<"OCA">>]},
       {qs,<<>>},
       {qs_vals,undefined},
       {bindings,[]},
       {headers,
           [{<<"host">>,<<"127.0.0.1:8065">>},
            {<<"upgrade">>,<<"websocket">>},
            {<<"connection">>,<<"Upgrade">>},
            {<<"sec-websocket-key">>,<<"r/TyIBb1ETjxdi+5kOD9fg==">>},
            {<<"sec-websocket-version">>,<<"13">>}]},
       {p_headers,
           [{<<"upgrade">>,[<<"websocket">>]},
            {<<"connection">>,[<<"upgrade">>]}]},
       {cookies,undefined},
       {meta,[{websocket_version,13},{websocket_compress,false}]},
       {body_state,waiting},
       {buffer,<<>>},
       {multipart,undefined},
       {resp_compress,false},
       {resp_state,done},
       {resp_headers,[]},
       {resp_body,<<>>},
       {onresponse,undefined}]},
  {state,<<"OCA">>},
  {terminate_reason,{error,handler}}],
 [{cowboy_websocket,handler_terminate,4,
      [{file,"src/cowboy_websocket.erl"},{line,752}]},
  {cowboy_websocket,handler_call,7,
      [{file,"src/cowboy_websocket.erl"},{line,641}]},
  {cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,442}]}]}
```

**Comment by devstopfix** [comment]
Game state:

```
 #{clock_ms => 1468238018024,
                          collision_pid => <0.323.0>,
                          explosions => [],
                          ids => <0.322.0>,
                          kby => #{<<"WDP">> => <<"SGW">>,
                            <<"SEC">> => <<"TSW">>,
                            <<"ILQ">> => <<"YGR">>,
                            <<"IZW">> => <<"EUT">>,
                            <<"BVJ">> => <<"YGR">>,
                            <<"OZI">> => <<"YGR">>,
                            <<"TJR">> => <<"VSI">>,
                            <<"RZE">> => <<"WHO">>,
                            <<"LHB">> => <<"FVU">>,
                            <<"DEB">> => <<"GJI">>,
                            <<"FPJ">> => <<"QXE">>,
                            <<"BMI">> => <<"AXR">>,
                            <<"GJI">> => <<"AXR">>,
                            <<"RAM">> => <<"MTS">>,
                            <<"HVH">> => <<"OZI">>,
                            <<"EBB">> => <<"SGW">>,
                            <<"ITG">> => <<"OCA">>,
                            <<"FUB">> => <<"MTS">>,
                            <<"SGW">> => <<"DEB">>,
                            <<"PRU">> => <<"FVU">>,
                            <<"WHO">> => <<"CXG">>,
                            <<"YGR">> => <<"FVU">>,
                            <<"WBE">> => <<"DEB">>,
                            <<"AXR">> => <<"WDP">>,
                            <<"FVU">> => <<"DEB">>,
                            <<"UUM">> => <<"QXE">>,
                            <<"WVW">> => <<"YGR">>,
                            <<"JNL">> => <<"SGW">>,
                            <<"EUT">> => <<"HVH">>,
                            <<"RKG">> => <<"GJI">>,
                            <<"RGG">> => <<"OZI">>,
                            <<"UDT">> => <<"YSW">>,
                            <<"POF">> => <<"LAO">>,
                            <<"MCI">> => <<"QXE">>,
                            <<"VSI">> => <<"TSW">>,
                            <<"OHZ">> => <<"HVH">>,
                            <<"ZAZ">> => <<"SGW">>,
                            <<"YSW">> => <<"OZI">>,
                            <<"LAO">> => <<"VSI">>,
                            <<"MTS">> => <<"UDT">>,
                            <<"RPU">> => <<"BMI">>,
                            <<"TSW">> => <<"EBB">>,
                            <<"CXG">> => <<"HVH">>,
                            <<"PDX">> => <<"EBB">>,
                            <<"QXE">> => <<"FUB">>,
                            <<"OCA">> => <<"MTS">>,
                            <<"AJV">> => <<"TSW">>,
                            <<"JKK">> => <<"QXE">>,
                            <<"SMC">> => <<"FPJ">>,
                            <<"VNV">> => <<"PDX">>,
                            <<"SIL">> => <<"TSW">>},
                          min_asteroid_count => 64,
                          pids => #{asteroids => #{29689 => <0.30209.0>,
                              29530 => <0.30050.0>,
                              29577 => <0.30097.0>,
                              29673 => <0.30193.0>,
                              29478 => <0.29998.0>,
                              29650 => <0.30170.0>,
                              29488 => <0.30008.0>,
                              29609 => <0.30129.0>,
                              29690 => <0.30210.0>,
                              29523 => <0.30043.0>,
                              29542 => <0.30062.0>,
                              29672 => <0.30192.0>,
                              29375 => <0.29895.0>,
                              29699 => <0.30220.0>,
                              29603 => <0.30123.0>,
                              28468 => <0.28988.0>,
                              29549 => <0.30069.0>,
                              29451 => <0.29971.0>,
                              29670 => <0.30190.0>,
                              29666 => <0.30186.0>,
                              29524 => <0.30044.0>,
                              29671 => <0.30191.0>,
                              29465 => <0.29985.0>,
                              29238 => <0.29758.0>,
                              29476 => <0.29996.0>,
                              29614 => <0.30134.0>,
                              29452 => <0.29972.0>,
                              29381 => <0.29901.0>,
                              29704 => <0.30225.0>,
                              29638 => <0.30158.0>,
                              29676 => <0.30196.0>,
                              29675 => <0.30195.0>,
                              29490 => <0.30010.0>,
                              29703 => <0.30224.0>,
                              29695 => <0.30216.0>,
                              29514 => <0.30034.0>,
                              29686 => <0.30206.0>,
                              29539 => <0.30059.0>,
                              28972 => <0.29492.0>,
                              29687 => <0.30207.0>,
                              29661 => <0.30181.0>,
                              29697 => <0.30218.0>,
                              29615 => <0.30135.0>,
                              29124 => <0.29644.0>,
                              29563 => <0.30083.0>,
                              29639 => <0.30159.0>,
                              29352 => <0.29872.0>,
                              29685 => <0.30205.0>,
                              29574 => <0.30094.0>,
                              29688 => <0.30208.0>,
                              29491 => <0.30011.0>,
                              29559 => <0.30079.0>,
                              29698 => <0.30219.0>,
                              29637 => <0.30157.0>,
                              29599 => <0.30119.0>,
                              29677 => <0.30197.0>,
                              29649 => <0.30169.0>,
                              29533 => <0.30053.0>,
                              29628 => <0.30148.0>,
                              29436 => <0.29956.0>,
                              29629 => <0.30149.0>,
                              29674 => <0.30194.0>,
                              29624 => <0.30144.0>,
                              28778 => <0.29298.0>,
                              29560 => <0.30080.0>},
                            bullets => #{29684 => <0.30204.0>,
                              29692 => <0.30213.0>,
                              29693 => <0.30214.0>,
                              29705 => <0.30226.0>},
                            ships => #{3081 => <0.3575.0>,
                              232 => <0.682.0>,
                              3046 => <0.3532.0>,
                              3045 => <0.3530.0>,
                              3043 => <0.3527.0>,
                              2989 => <0.3460.0>,
                              68 => <0.503.0>,
                              224 => <0.672.0>,
                              214 => <0.659.0>,
                              3025 => <0.3505.0>,
                              3002 => <0.3475.0>,
                              202 => <0.640.0>,
                              65 => <0.498.0>,
                              3040 => <0.3523.0>,
                              3050 => <0.3537.0>,
                              196 => <0.632.0>,
                              207 => <0.648.0>,
                              66 => <0.500.0>,
                              3012 => <0.3489.0>,
                              221 => <0.668.0>,
                              3055 => <0.3544.0>,
                              245 => <0.696.0>,
                              2996 => <0.3468.0>,
                              201 => <0.638.0>,
                              211 => <0.655.0>,
                              3077 => <0.3570.0>,
                              3087 => <0.3582.0>,
                              3104 => <0.3603.0>,
                              209 => <0.652.0>,
                              3067 => <0.3558.0>,
                              204 => <0.644.0>,
                              3020 => <0.3499.0>,
                              208 => <0.650.0>,
                              3061 => <0.3551.0>,
                              247 => <0.699.0>,
                              3090 => <0.3586.0>,
                              3034 => <0.3516.0>,
                              3009 => <0.3484.0>,
                              203 => <0.642.0>,
                              3016 => <0.3494.0>,
                              3010 => <0.3486.0>,
                              3029 => <0.3510.0>,
                              3095 => <0.3592.0>,
                              3006 => <0.3480.0>,
                              3053 => <0.3541.0>,
                              3073 => <0.3565.0>,
                              216 => <0.662.0>,
                              228 => <0.677.0>,
                              257 => <0.710.0>,
                              3103 => <0.3601.0>,
                              2969 => <0.3439.0>}},
                          state => #{asteroids => #{29689 => {29689,2384.3,1978.3,60.0},
                              29530 => {29530,3849.5,574.3,120.0},
                              29577 => {29577,1042.4,1205.3,15.0},
                              29673 => {29673,1854.6,1507.5,15.0},
                              29478 => {29478,2623.9,2030.5,15.0},
                              29650 => {29650,3830.5,1432.5,15.0},
                              29488 => {29488,3520.0,648.1,15.0},
                              29609 => {29609,124.5,468.8,15.0},
                              29690 => {29690,2256.6,2001.6,60.0},
                              29523 => {29523,3561.0,67.2,30.0},
                              29542 => {29542,275.4,730.2,15.0},
                              29672 => {29672,1970.6,31.8,15.0},
                              29375 => {29375,2659.6,1892.3,15.0},
                              29699 => {29699,3965.5,1356.7,60.0},
                              29603 => {29603,214.7,193.4,30.0},
                              28468 => {28468,3687.8,1558.8,30.0},
                              29549 => {29549,209.4,1625.0,120.0},
                              29451 => {29451,1948.5,1513.6,30.0},
                              29670 => {29670,3157.7,106.6,15.0},
                              29666 => {29666,15.5,545.0,15.0},
                              29524 => {29524,2711.4,837.0,30.0},
                              29671 => {29671,3024.3,1461.3,15.0},
                              29465 => {29465,61.1,1910.4,60.0},
                              29238 => {29238,3993.5,501.2,60.0},
                              29476 => {29476,2752.9,2065.5,15.0},
                              29614 => {29614,2172.8,479.8,30.0},
                              29452 => {29452,27.0,618.2,30.0},
                              29381 => {29381,3054.1,2106.8,60.0},
                              29704 => {29704,1049.6,1661.6,15.0},
                              29638 => {29638,252.7,2176.4,15.0},
                              29676 => {29676,302.1,1939.3,15.0},
                              29675 => {29675,3665.8,1953.1,60.0},
                              29490 => {29490,2998.2,1281.0,15.0},
                              29703 => {29703,1081.2,1807.0,15.0},
                              29695 => {29695,2498.9,135.6,15.0},
                              29514 => {29514,2681.5,2146.4,15.0},
                              29686 => {29686,3894.8,83.2,30.0},
                              29539 => {29539,3786.3,1199.3,120.0},
                              28972 => {28972,1237.8,1539.4,30.0},
                              29687 => {29687,1863.9,1116.7,15.0},
                              29661 => {29661,971.8,745.4,15.0},
                              29697 => {29697,2630.8,19.6,120.0},
                              29615 => {29615,2253.0,1820.0,30.0},
                              29124 => {29124,3869.8,573.7,60.0},
                              29563 => {29563,2273.1,1736.7,15.0},
                              29639 => {29639,175.7,1931.6,60.0},
                              29352 => {29352,68.5,1713.7,30.0},
                              29685 => {29685,3890.2,2082.9,30.0},
                              29574 => {29574,1022.7,954.3,15.0},
                              29688 => {29688,1560.8,746.9,15.0},
                              29491 => {29491,2959.1,680.9,15.0},
                              29559 => {29559,2691.4,243.7,15.0},
                              29698 => {29698,20.9,1358.5,60.0},
                              29637 => {29637,2847.2,212.3,15.0},
                              29599 => {29599,2284.0,1787.3,30.0},
                              29677 => {29677,3541.1,2023.0,15.0},
                              29649 => {29649,2627.0,732.5,15.0},
                              29533 => {29533,3556.2,199.2,15.0},
                              29628 => {29628,2764.5,1330.7,15.0},
                              29436 => {29436,230.1,1188.0,15.0},
                              29629 => {29629,1420.6,367.5,15.0},
                              29674 => {29674,3793.8,2121.2,60.0},
                              29624 => {29624,661.6,1444.0,15.0},
                              28778 => {28778,3966.4,1859.9,30.0},
                              29560 => {29560,2401.8,2013.9,15.0}},
                            bullets => #{29684 => {29684,1666.7,862.2},
                              29692 => {29692,752.1,1723.6},
                              29693 => {29693,532.8,106.2},
                              29705 => {29705,3846.1,1499.9}},
                            ships => #{3081 => {3081,<<"VNV">>,1637.8,1883.8,20.0,1.18899196,
                               <<"FFFFFF">>},
                              232 => {232,<<"EUT">>,3454.5,1974.2,20.0,2.66524155,
                               <<"FFFFFF">>},
                              3046 => {3046,<<"AXR">>,544.1,916.6,20.0,2.53194202,
                               <<"FFFFFF">>},
                              3045 => {3045,<<"AJV">>,565.3,681.8,20.0,2.98627997,
                               <<"FFFFFF">>},
                              3043 => {3043,<<"OHZ">>,632.6,1024.9,20.0,1.62236183,
                               <<"FFFFFF">>},
                              2989 => {2989,<<"JNL">>,3394.3,1295.2,20.0,0.62175344,
                               <<"FFFFFF">>},
                              68 => {68,<<"EBB">>,1131.9,342.4,20.0,1.46655919,
                               <<"FFFFFF">>},
                              224 => {224,<<"FPJ">>,2851.3,663.4,20.0,1.88654907,
                               <<"FFFFFF">>},
                              214 => {214,<<"TJR">>,451.0,272.3,20.0,0.33569233,
                               <<"FFFFFF">>},
                              3025 => {3025,<<"BVJ">>,1222.1,1285.4,20.0,1.72235817,
                               <<"FFFFFF">>},
                              3002 => {3002,<<"WBE">>,1750.4,1001.5,20.0,1.81297448,
                               <<"FFFFFF">>},
                              202 => {202,<<"YSW">>,2303.3,1265.4,20.0,3.13031494,
                               <<"FFFFFF">>},
                              65 => {65,<<"QXE">>,2237.7,1555.9,20.0,0.66638838,
                               <<"FFFFFF">>},
                              3040 => {3040,<<"IZW">>,1825.6,1929.0,20.0,2.72411683,
                               <<"FFFFFF">>},
                              3050 => {3050,<<"RZE">>,1885.0,1607.4,20.0,3.04686399,
                               <<"FFFFFF">>},
                              196 => {196,<<"ZAZ">>,2820.7,370.3,20.0,1.3504702,
                               <<"FFFFFF">>},
                              207 => {207,<<"HVH">>,2265.9,1036.5,20.0,1.40883732,
                               <<"FFFFFF">>},
                              66 => {66,<<"FVU">>,3498.9,967.8,20.0,1.88003373,
                               <<"FFFFFF">>},
                              3012 => {3012,<<"GJI">>,575.7,1527.3,20.0,0.19617691,
                               <<"FFFFFF">>},
                              221 => {221,<<"SGW">>,581.8,582.7,20.0,1.73578136,
                               <<"FFFFFF">>},
                              3055 => {3055,<<"SEC">>,441.8,222.3,20.0,1.3805004,
                               <<"FFFFFF">>},
                              245 => {245,<<"OZI">>,2809.6,1029.8,20.0,1.64706646,
                               <<"FFFFFF">>},
                              2996 => {2996,<<"JKK">>,621.1,313.9,20.0,1.79526719,
                               <<"FFFFFF">>},
                              201 => {201,<<"VSI">>,1712.9,1267.7,20.0,1.12450808,
                               <<"FFFFFF">>},
                              211 => {211,<<"SIL">>,1771.5,999.5,20.0,3.04708827,
                               <<"FFFFFF">>},
                              3077 => {3077,<<"RKG">>,1752.9,635.8,20.0,2.31607005,
                               <<"FFFFFF">>},
                              3087 => {3087,<<"RGG">>,3482.7,926.5,20.0,1.73543662,
                               <<"FFFFFF">>},
                              3104 => {3104,<<"PRU">>,3458.7,1594.9,20.0,0.74862894,
                               <<"FFFFFF">>},
                              209 => {209,<<"SMC">>,3413.0,1334.2,20.0,2.03458901,
                               <<"FFFFFF">>},
                              3067 => {3067,<<"LAO">>,479.1,675.5,20.0,2.86705201,
                               <<"FFFFFF">>},
                              204 => {204,<<"YGR">>,2346.9,654.1,20.0,2.93313909,
                               <<"FFFFFF">>},
                              3020 => {3020,<<"MCI">>,586.0,709.6,20.0,1.217784,
                               <<"FFFFFF">>},
                              208 => {208,<<"WVW">>,1698.3,1947.1,20.0,1.87016256,
                               <<"FFFFFF">>},
                              3061 => {3061,<<"OCA">>,1723.7,302.6,20.0,2.53619825,
                               <<"FFFFFF">>},
                              247 => {247,<<"PDX">>,1836.2,1941.2,20.0,0.33081375,
                               <<"FFFFFF">>},
                              3090 => {3090,<<"LHB">>,674.1,1309.0,20.0,1.12656189,
                               <<"FFFFFF">>},
                              3034 => {3034,<<"WHO">>,1096.2,261.3,20.0,2.48265557,
                               <<"FFFFFF">>},
                              3009 => {3009,<<"WDP">>,1681.7,859.6,20.0,2.99604706,
                               <<"FFFFFF">>},
                              203 => {203,<<"TSW">>,1686.7,1888.3,20.0,0.03207456,
                               <<"FFFFFF">>},
                              3016 => {3016,<<"CXG">>,1084.9,1626.9,20.0,2.25487739,
                               <<"FFFFFF">>},
                              3010 => {3010,<<"RAM">>,597.5,1312.5,20.0,1.52499872,
                               <<"FFFFFF">>},
                              3029 => {3029,<<"UUM">>,1757.3,1245.9,20.0,3.70694798,
                               <<"FFFFFF">>},
                              3095 => {3095,<<"ILQ">>,2192.9,239.2,20.0,2.68454803,
                               <<"FFFFFF">>},
                              3006 => {3006,<<"ITG">>,2927.6,364.8,20.0,1.62447514,
                               <<"FFFFFF">>},
                              3053 => {3053,<<"RPU">>,2766.2,360.7,20.0,1.20626687,
                               <<"FFFFFF">>},
                              3073 => {3073,<<"FUB">>,547.8,1279.4,20.0,0.14099611,
                               <<"FFFFFF">>},
                              216 => {216,<<"DEB">>,1667.6,1554.8,20.0,3.08503423,
                               <<"FFFFFF">>},
                              228 => {228,<<"UDT">>,1166.8,676.0,20.0,1.30294242,
                               <<"FFFFFF">>},
                              257 => {257,<<"MTS">>,2789.8,1288.9,20.0,1.64713671,
                               <<"FFFFFF">>},
                              3103 => {3103,<<"BMI">>,2186.3,330.5,20.0,2.52067873,
                               <<"FFFFFF">>},
                              2969 => {2969,<<"POF">>,517.3,1194.8,20.0,0.29399655,
                               <<"FFFFFF">>}}}}
```

**Comment by devstopfix** [comment]
```
** Reason for termination == 
** {{normal,{'Elixir.GenServer',call,[<0.30204.0>,{hit_ship,<<"WDP">>},5000]}},
    [{'Elixir.GenServer',call,3,[{file,"lib/gen_server.ex"},{line,564}]},
     {'Elixir.Game.Server',handle_cast,2,
                           [{file,"lib/game/server.ex"},{line,320}]},
     {gen_server,try_dispatch,4,[{file,"gen_server.erl"},{line,615}]},
     {gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,681}]},
     {proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,240}]}]}
```

**Comment by devstopfix** [comment]
We are only doing a `call` to retrieve who shot bullet. Could just send a message to the bullet (`stop`) which forwards info to Game when it stops itself

**Comment by devstopfix** [comment]
```
** Reason for termination == 
** {{normal,{'Elixir.GenServer',call,[<0.684.0>,{hit_ship,<<"UKR">>},5000]}},
    [{'Elixir.GenServer',call,3,[{file,"lib/gen_server.ex"},{line,564}]},
     {'Elixir.Game.Server',handle_cast,2,
                           [{file,"lib/game/server.ex"},{line,320}]},
     {gen_server,try_dispatch,4,[{file,"gen_server.erl"},{line,615}]},
     {gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,681}]},
     {proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,240}]}]}
```

**Comment by devstopfix** [comment]
Fixed in 1b9493bb
